### PR TITLE
kv (test): coverage of kv to 83%

### PR DESCRIPTION
**Pull request by jackysp**
Add unit test for kv. PTAL @zimulala @tiancaiamao

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

File: kv/fault_injection_test.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+type testFaultInjectionSuite struct{}
+
+var _ = Suite(testFaultInjectionSuite{})
+
+func (s testFaultInjectionSuite) TestFaultInjectionBasic(c *C) {
+	var cfg kv.InjectionConfig
+	err := errors.New("foo")
+	cfg.SetGetError(err)
+
+	store, _, err := newStoreWithBootstrap()
```
Comment (tiancaiamao) [inline kv/fault_injection_test.go]: Would `NewMockStorage` work here?
Function `newStoreWithBootstrap` is time consuming.

Comment (jackysp) [inline kv/fault_injection_test.go]: Ok, done.

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**